db.json

```diff
@@ -261,6 +261,17 @@
       "year": "2022",
       "poster": "https://lumiere-a.akamaihd.net/v1/images/p_drstrangeinthemultiverseofmadness_245_476cabb1.jpeg"
     }
+    {
+    "id":29,
+    "title":"Kantara",
+"plot":"Kantara is a mysterious forest and this is a story that happens around the area",
+"genre":"Action · Adventure · Drama · Thriller",
```

Comment:
Nice work!
Two changes though: 
1) Convert `·` into a comma `,`.
2) Keep indentation consistent.

Rest you are good to go! 🔥